WordPress/Classes/ViewRelated/System/Floating Create Button/CreateButtonCoordinator.swift

```diff
@@ -83,8 +83,10 @@ import WordPressFlux
         }
     }
 
+    /// Button must be manually shown _after_ adding using `showCreateButton`
     @objc func add(to view: UIView, trailingAnchor: NSLayoutXAxisAnchor, bottomAnchor: NSLayoutYAxisAnchor) {
         button.translatesAutoresizingMaskIntoConstraints = false
+        button.isHidden = true
```

Comment:
Defaults the Floating Create Button to hidden. This requires that the `showCreateButton` method _must_ be called before the button is shown. The only visual change is, when restoring to the Stats screen and hitting the "Back" button, the FAB now appropriately animates in.